src/App.css

```diff
@@ -214,9 +213,37 @@ h5 {
 }
 
 .json-block {
+  color: ghostwhite;
   height: 500px;
   overflow: scroll;
-  background-color: ghostwhite;
+  background-color: #2f3336;
+  padding: 5px;
+}
+
+#endpoint-input {
+  background-color: #2f3336;
+  box-shadow: none;
```

Comment:
The input feels small and hard to see, what if we added some styling like:
```
margin-left: 1em;
width: 300px;
height: 16px;
padding: 4px;
```
The color of the text while typing is very hard to see - I'm wondering about making that lighter too.

Reply:
Oh also I think lightening the placeholder text would be good!
```
input::placeholder{
  color: #fff;
}
```